azurelinuxagent/ga/exthandlers.py

```diff
@@ -1503,6 +1506,8 @@ def uninstall(self, extension=None):
         uninstall_cmd = man.get_uninstall_command()
         self.logger.info("Uninstall extension [{0}]".format(uninstall_cmd))
         self.launch_command(uninstall_cmd, extension=extension)
+        CGroupConfigurator.get_instance().remove_extension_slice(
```

Comment:
the cleanup of extensions files is done in remove_ext_handler below, we should move this call there

Reply:
Updated.